en/docs/learn/enterprise-integration-patterns/message-transformation/content-enricher.md

```diff
@@ -13,98 +13,183 @@ The Content Enricher EIP facilitates communication with another system if the me
-| Resource	                       | Local Registry                                               |
-| Enriched Message                | Enriched Stock Quote Request from the ESB profile of WSO2 EI |
+| Resource	                      | Local Registry                                               |
+| Enriched Message                | Enriched Stock Quote Request from the WSO2 MI                |
+
+## Synapse configuration of the artifacts
+
+Given below is the synapse configuration of this sample.
+
+=== "Proxy Service"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="ContentEnrichProxy" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+        <target>
+            <inSequence>
+                <switch source="//m1:symbol" xmlns:m1="http://services.samples/xsd">
+                    <case regex="1">
+                        <enrich description="">
+                            <source clone="true" key="Location1" type="inline"/>
+                            <target action="replace" xpath="//m1:symbol/text()" xmlns:m1="http://services.samples/xsd"/>
+                        </enrich>
+                    </case>
+                    <case regex="2">
+                        <enrich description="">
+                            <source clone="true" key="Location2" type="inline"/>
+                            <target action="replace" xpath="//m1:symbol/text()" xmlns:m1="http://services.samples/xsd"/>
+                        </enrich>
+                    </case>
+                    <default></default>
+                </switch>
+                <call>
+                    <endpoint key="SimpleStockEp"/>
+                </call>
+                <respond/>
+            </inSequence>
+            <faultSequence/>
+        </target>
+    </proxy>
+    ```
+=== "Local Entry 1"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <localEntry key="Location1" xmlns="http://ws.apache.org/ns/synapse">
+        <![CDATA[IBM]]>
+    </localEntry>
+    ```
+=== "Local Entry 2"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <localEntry key="Location2" xmlns="http://ws.apache.org/ns/synapse">
+        <![CDATA[WSO2]]>
+    </localEntry>
+    ```
+=== "Endpoint"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <endpoint name="SimpleStockEp" xmlns="http://ws.apache.org/ns/synapse">
+        <address uri="http://localhost:9000/services/SimpleStockQuoteService">
+            <suspendOnFailure>
+                <initialDuration>-1</initialDuration>
+                <progressionFactor>1</progressionFactor>
+            </suspendOnFailure>
+            <markForSuspension>
+                <retriesBeforeSuspension>0</retriesBeforeSuspension>
+            </markForSuspension>
+        </address>
+    </endpoint>
+    ```
+
+Let's investigate the elements of the configuration in detail. 
```

Comment:
If possible, shall we add the configuration explanation under each tab?

Reply:
We have added synapse configs in the tab, adding text will confuse. Therefore adding the description at the end of the synpase config section

Reply:
In the EI docs, we used a single definition file containing the proxy, endpoint and the local entries. In that case having a common description make sense. However since we have split those into different synapse configs, then we should at least mention which synapse config we are describing.